cropper/publish.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'maven-publish'
 
-def githubProperties = new Properties()
-githubProperties.load(new FileInputStream(rootProject.file("github.properties")))
+//def githubProperties = new Properties()
+//githubProperties.load(new FileInputStream(rootProject.file("github.properties")))
```

Comment:
Instead we comment this file, could we comment `apply from: file('publish.gradle')` in the `cropper/build.gradle` ?

And add this comment:
```
// publish.gradle is used for release process, until we can automate using the CI, pease leave it here. @Canato
// apply from: file('publish.gradle')
```

Reply:
Sorry, this was not intentional. Updated.